py3status/modules/thunderbird_calendar.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Display tasks in thunderbird calendar.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh usage in seconds (default: 120s)
+    - profile_path : path to the user thunderbird profile (not optional)
+    - format : see placeholders below
```

Comment:
yes,

if you look in master branch all the modules now conform so you can check with them.  They get transformed for the README.md

basically
```
One line desc

general blurb

Configuration parameters:
    cache_timeout: how often we refresh usage in seconds (default: 120s)
    profile_path: path to the user thunderbird profile 

Format of status string placeholders:
    {due}: due tasks
    {completed}: completed tasks

@author me
```